cns/kubecontroller/clustersubnetstate/reconciler.go

```diff
@@ -22,8 +22,10 @@ type Reconciler struct {
 func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reconcile.Result, error) {
 	css, err := r.Cli.Get(ctx, req.NamespacedName)
 	if err != nil {
+		cssReconcilerErrorState.Set(float64(1))
```

Comment:
same here, if you have exported `const`s you can use them here and it will be clear what you are setting this to